README.md

```diff
@@ -194,6 +194,11 @@ otherwise terraform will detect changes to the record when none actually exist.
 
 A record _requires_ a [Zone](#zone)
 
+The **zone** and **domain** fields should not have any leading or trailing dots (".").
+If the value is coming from another resource with a leading or trailing dot, it should be cleaned:
+
+`zone = replace(".terraform-test-zone.io.", "/(^\\.)|(\\.$)/", "")`
+
```

Comment:
I might omit this, or just mention add "and the zone should not have leading or trailing dots" to the sentence above, and focus documentation details on the record doc. feels a little edge-casey for the kind of broad summary here.